modules/virt-about-runstrategies-vms.adoc

```diff
@@ -4,28 +4,52 @@
 
 :_content-type: CONCEPT
 [id="virt-about-runstrategies-vms_{context}"]
-= About RunStrategies for virtual machines
+= About run strategies for virtual machines
 
-A `RunStrategy` for virtual machines determines a virtual machine instance's (VMI) behavior, depending on a series of conditions. The `spec.runStrategy` setting exists in the virtual machine configuration process as an alternative to the `spec.running` setting.
-The `spec.runStrategy` setting allows greater flexibility for how VMIs are created and managed, in contrast to the `spec.running` setting with only `true` or `false` responses. However, the two settings are mutually exclusive. Only either `spec.running` or `spec.runStrategy` can be used. An error occurs if both are used.
+Run strategies for virtual machines (VMs) determine how virtual machine instances (VMIs) behave under certain conditions.
 
-There are four defined RunStrategies.
+You configure a run strategy by assigning a value to the `runStrategy` key in the `VirtualMachine` manifest as in the following example:
+
+.Example run strategy
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+spec:
+  runStrategy: Always
+  template:
+# ...
+----
+
+[IMPORTANT]
+====
+The `runStrategy` and the `running` keys are mutually exclusive. Only one of them can be used.
+====
+
+The `runStrategy` key gives you more flexibility because it has four values, unlike the `running` key, which has a Boolean value.
+
+.runStrategy key values
```

Comment:
Backticks are allowed in `.` titles (usually block titles, but the point I believe is that it's not a section title that would appear in a TOC), so you can mark up `runStrategy` here if you like.

Per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references):
> Do not use backticks or other markup in assembly or module headings. You can use backticks or other markup in the title for a block, such as a code block .Example or a table .Description title.